borgmatic/actions/create.py

```diff
@@ -1,15 +1,51 @@
 import json
+import os
 import logging
 
+import importlib_metadata
+
 import borgmatic.borg.create
 import borgmatic.config.validate
 import borgmatic.hooks.command
 import borgmatic.hooks.dispatch
 import borgmatic.hooks.dump
 
+from borgmatic.borg.state import DEFAULT_BORGMATIC_SOURCE_DIRECTORY
+
 logger = logging.getLogger(__name__)
 
 
+def create_borgmatic_manifest(location, config_paths, dry_run):
+    '''
+    Create a borgmatic manifest file to store the paths to the configuration files used to create
+    the archive.
+    '''
+    if dry_run:
+        return
+
+    borgmatic_source_directory = (
+        location.get('borgmatic_source_directory')
+        if location.get('borgmatic_source_directory')
```

Comment:
I don't think this `if` is necessary since `.get()` returns None if the key is not found...?